lib/fog/aws/requests/rds/create_db_snapshot.rb

```diff
@@ -26,7 +26,7 @@ class Mock
         def create_db_snapshot(identifier, name)
           response = Excon::Response.new
           if data[:snapshots][name]
-            raise Fog::AWS::RDS::IndentifierTaken.new
```

Comment:
Thanks!

On Wed, Mar 23, 2016 at 11:17 AM, Josh Lane <notifications@github.com>
wrote:

> In lib/fog/aws/requests/rds/create_db_snapshot.rb
> <https://github.com/fog/fog-aws/pull/238#discussion_r57189046>:
>
> > @@ -26,7 +26,7 @@ class Mock
> >          def create_db_snapshot(identifier, name)
> >            response = Excon::Response.new
> >            if data[:snapshots][name]
> > -            raise Fog::AWS::RDS::IndentifierTaken.new
>
> lulz
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/fog/fog-aws/pull/238/files/8cb360ca78df14a156055d0f62dae2cd24011788#r57189046>
>